mkt/fireplace/views.py

```diff
@@ -31,6 +31,7 @@ class MultiSearchView(BaseMultiSearchView):
     serializer_classes = {
         'extension': ESExtensionSerializer,
         'webapp': FireplaceESAppSerializer,
+        'homescreen': FireplaceESAppSerializer,
```

Comment:
Add a test showing that homescreens are included when requested in `mkt/fireplace/test_views.py`